po/da.po

```diff
@@ -675,34 +675,32 @@ msgid "With Java"
 msgstr ""
 
 #: src/SUMMARY.md:207
-#, fuzzy
 msgid "Bare Metal: Morning"
-msgstr "Dag 1: Formiddag"
+msgstr "Rå jern: Formiddag"
```

Comment:
nitpick: "bar metal" might be easier to map back to English. I found this usage on the web:

> Hostet bar metal fremstår som alternativ til IaaS-sky (Offentlig sky...

I don't know which translation causes the least mental overhead - either is good with me.